packages/cli/src/commands/setup/deploy/deploy.js

```diff
@@ -50,6 +50,13 @@ export const builder = (yargs) => {
       description: 'Overwrite existing configuration',
       type: 'boolean',
     })
+    .option('database', {
+      alias: 'd',
+      choices: ['none', 'postgres', 'sqlite'],
+      default: 'postgres',
+      description: 'Set Prisma database provider',
+      type: 'string',
+    })
```

Comment:
@peterp @dac09 Could use some thinking about how to handle this top-level option for `rw setup deploy <provider>`. Any reactions/ideas about whether or not there's a better way to handle this?
- for the time being, it's only needed for Render deploy setup
- it's currently visible/presented as an option for all providers (see code at bottom)
- if we could hide for `rw setup deploy --help` and _only_ display at `rw setup deploy render --help`, I don't think will be obvious. Not a recommended solution.
- currently, it is ignored when passed for other providers — no error issue. It's just possibly confusing

**Ideal Handling:** can we can make it clear the --database option _only_ applies to Render (at this time). At a minimum, we could update the description. Also, I'm ok if you all don't see much that's concerning.

Current --help output:
```bash
rw setup deploy <provider>

Generate a deployment configuration
Positionals:
  provider  Deploy provider to configure
           [string] [required] [choices: "aws-serverless", "netlify", "render", "vercel"]

Options:
      --help      Show help                                           [boolean]
      --version   Show version number                                 [boolean]
  -f, --force     Overwrite existing configuration   [boolean] [default: false]
  -d, --database  Set Prisma database provider  [string] [choices: "none", "postgres", "sqlite"] [default: "postgres"]
```

Reply:
This seems fine to me, but in my mind I would imagine that a user would do this outside the setup deploy command.

Reply:
Because Render provides "infrastructure as config" along with the Jamstack deploy process (i.e. push to GitHub triggers build + deploy), this is the first time a Redwood deploy target has been able to offer Web + API + DB hosting. Given that being valuable to devs, I want to make sure users have access to including DB at the time of setup. As a next-step experiment, this feels like a good step.

However, it will become more complex when/if more infrastructure options are available. And/or what if I want to "setup" my DB but use the Serverless framework to do so on AWS -- should we make that easy too and then would we create a separate `yarn rw setup database` command... ?. Not advocating we do so. Just saying I see complexity down the road.

I'm ok to leave it as is for now if there's nothing else obvious as an alternative. We can improve and iterate as needed.

Reply:
@thedavidprice I spent quite a bit of time digging into yargs and seeing if I could make the database option conditional on selecting the provider to be Render to no success. I think the longer-term fix could be to make the CLI more interactive like we talked about to avoid having to pass in this option in.

The lowest hanging option here is to add that as a note for the -d flag. For example; "Set Prisma database provider, for Render only"

Reply:
Could you also add a [`check()`](https://yargs.js.org/docs/#api-reference-checkfn-globaltrue) that prints a helpful message if someone tries to use `--database` with something that's not render? (IIUC positionals are available in `argv._`, so something like `if (argv._[0] !== "render" && argv.database) { /* show error */ }` perhaps)

Reply:
@Tobbe I was curious about the same and ran some tests locally to get a gut feel. It's all silent and in the background. So great idea, but I don't think it would add much value at this point assuming 1) we make it clear in the CLI doc 2) we update the description

@SEANDOUGHTY yes, please change the description to something like that "...Render only option"

Reply:
✅